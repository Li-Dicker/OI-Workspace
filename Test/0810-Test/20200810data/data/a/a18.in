10
197
78022 39834 61083 74400 13157 11641 21155 24445 81865 58888 98711 23204 77150 85707 90971 80723 93930 57840 44497 16388 8405 16857 90572 94271 43446 29120 2754 4219 93956 77101 22861 48278 18059 78707 9869 45881 87538 88496 82479 36899 87183 99154 41775 72362 18753 99566 84180 41627 75012 71976 11299 71273 61034 80917 73736 16847 43667 7909 21900 16966 63434 62132 26930 83092 47635 82079 51513 30054 32094 14902 58739 48706 15231 1952 71841 10181 58640 71339 55115 53492 14860 54337 78465 40852 67225 84448 80490 74466 96933 97667 96640 84358 8434 75852 78462 85395 25677 76196 19820 75746 66842 58423 49885 13891 50489 19077 81031 95861 6227 28600 56983 63109 51192 77192 51850 37580 91319 42747 10538 88129 956 92814 23911 83906 44874 54030 95957 34505 67015 46578 32219 75897 2156 80375 34432 68393 95489 18697 40014 46191 83929 80852 85395 26906 39800 93053 96587 98724 82974 87470 19507 77638 90818 2499 51495 1073 44567 81581 22745 27815 33953 55618 24195 70909 56357 20609 36659 31772 67475 85375 3085 27109 76816 26260 24642 83237 47808 10332 82385 76261 4850 35593 10465 67374 98696 16104 43562 80607 25110 27040 82387 53079 5174 45951 80395 20941 7052 
43504 29970 39091 44148 25644 6213 9529 45131 48576 921 26143 26261 48206 20928 31179 11916 25734 24282 42386 42533 8711 19269 32344 49862 25978 10649 3730 23516 49071 13544 33132 33810 35650 41295 39682 30401 24445 5651 34107 14407 38725 27378 28763 8098 26312 23634 31466 38424 11731 3127 37253 20434 5895 35823 32537 8783 41996 11310 15611 4934 39008 32905 26857 25228 44509 18785 29285 17039 5394 42016 6445 30564 33391 18881 16228 39979 21419 27511 48257 41648 27655 3134 42779 27346 12885 45374 47674 33213 21584 15460 32944 40540 27096 16372 35995 15786 5474 22095 11577 34395 39009 17761 18459 869 18999 44051 27716 1901 31658 47044 40032 45753 45760 48821 30571 27066 39897 38607 48601 26630 34966 31420 3531 34493 35913 19827 38657 35165 10427 9209 18443 35964 313 24270 39991 43617 16748 19834 15122 3210 8253 20212 46650 27996 36578 44330 440 33417 39514 29696 49671 45607 16451 47463 12735 3347 23610 8480 17792 16130 33715 7016 17673 8313 33291 10879 8141 28821 17880 35202 9690 25320 36918 8384 7940 47010 8247 39017 11578 49199 10903 24862 20031 40507 45563 21905 36591 46451 7949 39912 49814 16314 18637 37613 12921 6638 25385 
198
95340 47661 97223 543 49539 16149 98929 45403 89674 56828 20784 53121 85546 65961 10131 39364 36742 26797 24681 45880 27701 45284 16846 30304 70737 4115 17095 18157 45636 82269 18066 23483 12306 56782 29938 12391 3936 25431 8901 47110 87306 16671 93035 34085 43162 12103 97711 20793 52353 10845 95844 24356 84378 98048 57643 64219 72427 33361 70097 98982 67247 37732 52188 94973 49769 57328 15324 19224 73888 22368 19518 40594 9929 58756 78160 93056 61159 10533 91286 96875 89442 69814 86293 54409 65955 38523 24906 15598 49603 32072 89929 16451 4734 50965 71017 56083 79323 54837 52869 3872 16649 16230 81463 45183 64311 30541 50342 53106 28595 45224 17928 25499 89958 20797 36466 90918 1738 56464 75833 10285 84459 23139 53910 69069 24266 46905 4776 38437 47738 9062 99645 29396 62965 63760 32418 31062 37479 23592 65692 52006 83694 78702 14360 13700 12244 64531 44415 44662 34046 13433 90753 57731 72202 73536 26346 70016 79000 61809 6791 88350 95525 61689 18994 18150 70090 77604 95359 17558 68967 50919 60564 42218 84429 57114 9336 16006 3309 11354 81662 10030 24950 31953 833 56069 93346 2788 92978 78269 61134 87846 71125 35763 537 98500 97380 426 6609 50652 
77 26757 35204 32274 33923 48923 2782 41940 15647 29869 15767 8687 46076 38785 19321 41914 21013 19579 20753 1033 37962 24187 39762 49470 43662 27664 3069 39352 23181 3493 17122 35512 49524 20401 19285 36763 32767 18130 28757 14383 14409 27555 47848 229 20376 31343 12184 13223 7756 29949 32282 42260 29511 40836 23774 14032 16090 2519 31081 49833 44559 29529 11777 26746 44056 3031 40564 17916 45264 1379 14673 16856 26173 30990 13412 33551 48135 46587 4213 25373 12991 24519 23548 19326 17435 31572 8562 12306 28026 9368 13054 18879 30055 47275 26070 29413 19124 33368 43110 48260 5149 16701 32540 17079 1891 3348 48239 24632 311 22206 26359 16070 18378 3407 25206 19619 285 39957 13717 9588 38723 47966 23048 22798 1325 7866 7179 30497 8786 49807 2041 42738 951 42821 2642 37560 34283 5804 25715 475 35143 38315 22012 13694 24208 49473 39930 19781 48684 40929 10549 33901 18459 47743 38261 28242 23818 18646 4007 9076 1611 45534 8314 49042 31942 46072 11106 30483 834 45347 8203 28759 40820 2951 31470 44906 38676 17646 9373 23013 9481 97 43484 25706 1652 38106 20335 516 17489 29051 9251 20788 47190 1014 32557 718 39766 6875 
197
39467 15938 69758 66665 70015 99102 51837 31040 27684 59712 37702 44342 48974 82881 44033 19808 22144 29146 91416 17159 64443 41789 50628 81558 97228 98941 44827 86258 70895 96355 69584 74706 12031 74931 70479 26538 33894 46118 36872 92739 48874 27644 10360 71483 16572 11549 44786 61601 77925 1405 6035 36526 37506 31398 69650 14354 5848 46192 71669 66043 55487 78596 16156 98686 40380 92294 98093 68542 25699 18085 45085 75899 54933 82139 93066 51822 82838 83215 96666 59512 9523 18968 83690 59857 67854 83195 68649 86548 67427 24660 51566 83643 76653 67690 13667 56122 16139 90091 41977 91296 88490 68744 81160 35184 66385 32978 62917 37239 9608 72091 92060 90373 83964 44859 89170 25069 34451 83447 63866 97127 3942 55673 74858 62985 33832 37629 95469 37072 22953 46283 49109 73682 42181 99317 97240 50053 80619 11368 16755 27090 35607 34355 31312 85205 93099 96325 43180 58733 3234 15543 45049 92541 55432 9731 12130 9924 13452 25043 76840 41259 17966 53528 7168 14304 93689 27399 98010 37325 11419 96007 2276 74932 91603 8077 79104 22521 3952 8665 76066 57547 18705 67451 57844 75156 83381 6481 53724 89595 83866 7302 89454 24290 81475 50280 26987 23133 99073 
39353 11646 20093 37799 30219 36722 25502 19785 49545 29931 28250 26288 45522 25763 31355 21733 21587 10871 17587 41827 40638 39537 23299 7412 19500 42038 579 25851 18690 48187 41727 34190 26201 39027 40687 38491 37569 47920 18321 2925 33846 26502 4787 35882 17175 36032 34768 10492 2490 40621 42738 39068 43875 46160 43301 10418 8636 10068 44720 13006 45017 46496 1667 35991 7011 32234 130 33446 10302 199 46465 12314 8160 21201 39388 8325 46278 37080 30129 30831 12879 44918 24496 18073 41466 35681 28459 5479 3640 18370 13665 19821 16048 46708 41674 38951 6981 17674 23975 9747 5696 23739 37864 35411 28699 46282 44849 48970 44257 817 2636 15338 16231 863 6343 41460 42755 46907 49471 41752 34887 34409 47444 35108 23557 37292 35220 26357 40615 45193 27027 44828 40306 28391 45308 14468 31346 6066 10476 34906 24211 33644 14045 46301 29446 6213 42785 100 43274 7617 34987 7697 23425 19389 34690 14940 3389 22228 41644 48423 17576 49127 8305 33371 43125 35033 37099 13480 28548 39044 39440 7370 24104 15400 46436 48375 22233 8418 36932 44196 10299 26971 24355 10076 49628 4891 48198 45015 47322 17939 22003 19784 35136 34136 34035 31090 36943 
196
10251 25981 21134 14035 20957 78504 53979 43265 97486 15539 4068 57421 60297 34137 43779 1845 76574 46642 38565 90531 87359 7298 5571 78877 4359 58910 31787 28725 21700 2066 44166 99455 39444 79380 13346 88431 81864 85703 18483 52967 76154 31137 98643 60631 2493 64528 94793 65663 10347 63936 2601 86772 64924 45649 96615 95094 12877 35168 92930 10327 9236 62473 42046 83371 10588 95885 58548 5721 89209 12774 42353 7048 17 24751 40936 98804 97834 1131 8182 76625 46946 66162 42038 80584 17217 37454 91053 4048 67340 26332 31308 98313 44109 10489 45016 46768 82297 90460 43484 15501 70884 94819 78236 40776 69612 48227 56239 53342 43842 79752 76837 5184 89323 42011 53495 60842 49223 20918 48046 81974 38650 95801 9799 20121 4313 93492 90293 4424 98485 25544 36824 45930 72854 45469 1910 69664 29573 66405 84553 28794 92264 23195 75576 38169 87891 63803 84435 36195 49566 55713 17107 19796 48537 18293 43834 54032 48108 6321 93658 49684 90052 78275 59763 77706 62519 30895 66026 25601 16580 31112 49487 75491 7854 31 651 83858 14515 51332 93650 93839 85405 80059 29674 23152 5748 50470 99666 24108 84577 25762 92345 62875 30305 81684 84349 45727 
41723 12074 6728 41330 3370 164 18042 9138 22577 25565 34427 8058 32839 32457 18408 41180 46185 44081 38746 49919 47236 33378 37920 21838 14388 26202 8891 4621 25286 6793 1152 22193 3863 23108 40298 19036 30246 21543 9937 2507 9594 34091 32918 6453 38129 37403 24377 32520 19323 39 40845 13432 26041 47050 41878 14247 17754 23400 37173 21215 37611 8213 39165 37830 38188 47494 32930 8846 46536 21900 43658 10848 20884 32431 40242 9679 18897 40703 6610 39336 17817 33290 510 30731 26379 5001 27554 579 7874 33637 24169 27792 8909 43621 41594 22711 41377 48634 37507 42062 6812 6642 14161 26166 5884 18092 41685 40795 40074 44335 19007 1572 33139 48584 43369 16579 36283 44734 17136 4884 31766 36188 40328 41873 42389 46763 42688 19179 22831 28862 2397 7612 12225 18503 30498 24628 25055 17932 790 14901 44845 20323 26488 49852 3337 36579 21603 12301 49999 20680 33117 46423 7097 29235 10940 12352 2283 46873 23926 46133 46464 34348 24249 30565 35997 19125 12460 29587 46761 41877 20079 36407 24724 3415 39185 43048 2654 48912 7723 49897 46260 43807 25638 17896 34002 24820 6455 39520 27687 41812 35091 19379 16861 19122 40433 20198 
197
70504 75741 4993 88541 77469 65631 96865 39078 27787 22686 49135 68417 645 20185 21338 7154 25880 6537 58006 67746 6593 26130 67307 99918 78144 91772 28587 19653 4497 54946 24526 51825 81573 25972 53482 58909 7312 75104 55618 14498 29347 64215 47671 25984 80415 23377 37984 75042 75392 61318 61036 76231 59945 45102 669 37482 7382 49925 75716 49525 44356 57056 90665 59642 24839 36173 77439 5424 55610 23828 45856 93363 59114 46685 2486 99573 36554 32549 4244 40675 96045 56985 61100 74298 66799 22991 30645 76436 68235 7414 22268 17400 65137 24446 34314 11223 2059 23135 71571 13258 17989 71611 59517 59270 77160 36766 23720 55318 90221 93532 96402 14884 37315 51297 59417 51920 70467 24422 59050 15316 28970 4644 60533 32060 23904 48970 39951 56667 38122 16649 79743 35994 40899 91270 27578 89547 58254 57031 79089 74036 46183 30704 74344 88731 28910 80559 75284 77634 25613 68220 26738 63977 5321 69004 30293 13602 97475 64885 72283 62459 70931 97363 62212 22658 97303 42594 43658 560 62468 3542 52701 87399 25599 29406 9294 38578 84162 69760 78816 34671 41103 86240 30466 88184 34469 13186 53245 5257 53030 24518 84700 13162 23692 95082 95613 824 61372 
47943 14362 45525 10449 22103 10561 44267 19129 39564 3307 35351 38201 30068 16013 16601 45666 5799 36208 39975 43995 33234 42804 19378 49126 11038 48844 501 41212 41211 4398 9422 17015 24947 41711 14722 32100 10965 5598 29578 19460 40755 22156 41851 34055 39940 30796 1019 42809 18714 1942 38852 24387 48745 30163 25403 38121 43622 25216 13808 46527 30562 20044 13115 37768 19410 27726 36282 44942 12066 9792 4299 29956 34882 20 786 18775 30457 27657 23692 3827 45556 34301 29871 44067 40219 37384 48531 23985 3567 19242 49968 3323 43219 36758 14479 28436 19115 31752 14544 40019 1098 5215 27965 9575 13877 25559 22674 3418 6954 20772 37582 49763 45151 49838 21466 1011 21954 17586 13104 41260 49317 42326 22981 21938 2166 16621 39222 38707 27353 49321 17989 5283 32058 28452 16593 23205 12042 3269 6681 23856 47598 35667 19093 23399 18371 15129 48352 35569 37596 26979 29660 43393 17288 49659 45458 40685 16527 48279 9959 43740 14670 27359 44726 17668 38567 30224 69 47039 33728 31439 36461 44263 7722 11682 43238 23728 44393 35733 22077 30180 10464 4567 41473 38341 26973 10577 41030 32318 18586 6230 44466 8760 10893 11398 25652 15123 32226 
198
99499 23628 65421 27329 76786 98362 50267 87251 41215 47284 72241 83575 31203 8177 59023 60085 92337 7144 42836 90291 49646 63320 5854 35565 66385 75089 47288 82069 80903 33013 66415 1995 58417 73046 13527 98694 37813 64866 15575 41651 1676 45562 94124 55971 86710 85113 33651 68889 30937 54244 16128 74677 95600 35962 52932 77581 23978 99914 32672 4389 59676 56560 4189 166 59882 38055 86320 38902 85701 31191 52325 51753 82412 66505 45456 589 17517 56013 73971 2631 5308 87148 12245 41586 49785 14968 14115 39985 8450 19664 232 80194 20778 41459 3211 86650 63367 58120 51407 7119 31145 38851 87769 52725 7483 55340 66861 87431 3936 95880 42422 61072 12539 82908 49896 92827 91571 41597 24855 31692 60495 49924 467 22128 47659 75951 35797 38312 5226 84009 6287 85623 65071 11274 31091 25530 56851 34180 53193 97812 6544 66702 85054 65418 67758 36630 55726 82725 57905 19760 2514 10776 95978 21000 70879 19721 46916 62987 52813 87922 8802 58565 20269 74779 62030 21043 4487 70890 52897 79342 19047 34501 7349 12378 29842 51557 36968 94282 82892 90440 33216 25180 84243 84457 14389 55231 27515 52834 5378 84069 61041 8966 12131 1466 18557 20720 76803 58818 
13010 19750 6444 21667 5534 28875 44121 893 8504 41095 40441 43872 8433 2951 39806 36175 28533 39699 45677 44225 8258 14086 10443 19712 16633 12568 2233 22533 40705 26738 39521 24813 16522 23660 25587 30052 38913 32253 43458 39938 33856 3831 45872 33868 4846 2025 4037 5337 40704 20524 35827 11140 24774 38149 24258 6760 22047 2540 30479 35212 35886 7928 44947 16908 41029 12895 36388 6055 49323 24513 27420 32960 47864 17363 38634 8056 20239 40093 18737 8675 12276 21393 19691 32631 27071 33381 35442 41640 15229 29402 2580 31466 41219 19711 41519 25993 1982 12763 34666 48393 47388 11987 32183 42471 18736 19051 9369 44493 33526 1409 43182 19388 29750 20004 22018 23971 38887 14745 4611 22351 16782 3257 27111 1045 34945 9456 9841 30778 8632 241 24147 8918 1639 10616 17639 40634 5360 29330 34910 38438 1877 2971 35208 2766 28234 8200 17579 48711 37381 6809 31358 48012 454 35940 20880 47840 15581 20021 565 5181 14012 25801 41300 9813 38235 17432 19963 11727 49413 31075 20014 17943 13510 21101 21326 2355 11461 45030 4588 78 16642 18303 21482 49389 49271 4785 42460 47482 3438 12890 19130 10680 22555 2302 13317 40466 44359 43559 
197
80156 7643 86725 50440 1677 90805 45305 46766 9913 8677 1607 54295 8030 70118 59563 6992 51739 30553 29210 74451 43321 53386 57907 37365 28512 81044 55068 4519 67606 32280 18074 48503 76678 22645 77010 40985 13638 45623 22024 28 50310 72600 64673 56193 6160 96938 35677 49245 97523 45498 77081 99086 31549 79558 2907 43420 19992 36461 56623 58086 33445 23567 66500 30626 57746 99177 69316 85441 27006 55364 22608 32765 50396 34839 8864 81070 96244 15214 97143 13505 79057 39113 73075 75712 30872 80635 54925 75640 30914 7946 26740 95480 9474 85686 62587 9765 84069 98751 5727 5500 17794 14931 99387 63904 24136 46312 98440 36647 777 71401 21835 37494 70290 91226 3619 4813 59948 59702 8949 59464 93557 54612 5086 70443 28084 47253 21798 33515 36112 76621 85927 8164 40547 66680 57227 89239 74167 48300 81262 86545 63728 69543 98953 39176 90968 60229 921 1586 47071 70689 74651 38390 44763 58687 83884 79138 44798 41481 44903 26356 76264 30448 14170 80680 2366 8877 81225 27693 83351 29545 88833 71264 50495 19962 71908 59646 46126 20890 16958 95923 41984 13300 50396 3395 42825 49893 30506 80367 29834 367 12900 37563 84989 79342 53994 638 95641 
8033 42337 44811 8103 16162 23765 48707 34981 38596 41098 3837 14689 11142 3012 19787 31698 46009 8789 25996 31717 8070 21938 12499 12404 23145 18401 48228 5471 17370 40473 17678 26011 30713 6468 8791 24008 44907 26953 22365 8058 33868 6893 13800 3810 21654 25329 20392 38013 18043 25753 25868 39714 10610 7583 1075 28182 14793 18887 20306 1191 13273 25126 14338 45126 18388 15004 17156 47762 5981 23632 34845 23131 18878 2905 35737 34633 5970 19876 48283 18943 11792 2560 25309 40848 15640 21113 47126 43617 36018 15404 36759 48021 22473 24713 38967 34478 41764 23321 36123 46486 4907 30554 34114 7605 43318 46262 2762 42869 6858 23420 12089 29512 39028 40468 38118 26680 5964 36207 32362 5462 37535 11940 3163 32371 2552 8042 16424 34251 15007 9905 33304 46905 26882 39607 40164 12047 7178 21627 27336 32861 10548 47536 26653 43367 8709 9637 10882 13634 1941 47152 49240 33916 15221 45556 32959 14094 1487 20483 49832 26722 890 25596 15615 38184 16428 36195 46147 26282 851 16757 6689 8924 9808 7216 40974 15530 40540 28559 48103 19804 30141 15210 15354 13312 2559 29588 45840 48533 29974 35329 10933 48960 21512 25464 33436 29817 2510 
196
99181 63672 35313 27227 25401 63540 77505 14003 63677 25877 81275 97562 49536 21126 82228 99577 74774 96486 65067 65966 16427 71071 75316 30133 33689 62199 85532 78651 84084 34989 34512 91574 28625 59480 13490 16330 8213 66761 74025 43947 37694 58848 48402 37366 88775 64582 17051 4082 47683 88764 58105 72176 23184 56993 62520 38368 2264 88884 22921 11012 99132 95658 41724 79868 93070 28824 60226 24391 46106 79198 89153 21078 52461 37119 9769 7928 23152 63570 83234 58759 26073 39268 28828 52472 3150 58201 48237 26211 46294 95070 95117 87223 32874 41425 57190 39789 37573 45167 56667 67444 48364 22145 12824 16018 47018 98743 85072 68920 36646 48538 71433 41704 59822 90119 28740 68926 75486 73095 86020 10306 85289 94223 35069 37308 39641 73096 10852 16425 85746 53780 7080 17454 7861 29910 46298 86389 88582 1508 82672 27475 90099 88981 88699 75113 13501 1537 37532 1682 74045 47541 57609 15266 56833 17878 21510 76598 60762 6181 54086 17520 75893 62575 89237 61066 68330 25464 38304 33897 52336 16221 56731 41568 65789 80880 89545 5997 5906 50108 79000 74989 50740 9346 57761 61111 45990 48401 15099 40176 48884 39806 52977 91522 69466 54649 9098 34098 
35786 46727 41548 37883 49722 16820 31358 10135 41771 40323 49352 14227 28896 8687 10013 49054 35778 34532 19082 19129 33678 5678 27911 7155 43008 24598 21656 2232 30828 25340 47695 1403 2784 9745 46957 49563 9515 13042 20303 5863 20683 5258 7704 29788 13424 38805 30865 35834 25034 7963 39677 5165 12461 48052 5532 36464 14231 35980 44255 36833 36560 21181 19455 8950 36655 30312 42087 25723 4990 36741 3338 47782 10355 43567 42169 38161 12192 10370 6846 22843 6904 29606 4532 32386 40942 42719 23148 41775 30938 28284 19232 39234 13993 22222 24696 46522 940 41091 23238 35272 9002 1304 48495 23435 23691 29339 103 34479 25648 20876 31881 47264 28399 39004 32696 37956 14389 22694 16718 44375 21176 1465 41530 32127 17388 17349 44202 14414 4395 14809 25691 25374 37866 17027 1730 17224 17665 11807 19546 13844 8744 33372 12614 7302 2363 13029 25678 36969 48446 16203 26714 46646 37799 43035 34346 40964 38560 40664 43248 13229 10564 26201 44052 43218 7922 11414 9028 35623 9995 19877 24390 40320 46394 17120 24319 44627 32040 1947 30480 22496 8731 29729 13932 15058 4038 49826 20901 26016 39699 7584 919 1196 18849 45208 22899 48385 
200
55556 74923 91573 75056 23219 27419 49352 43095 79624 53660 23991 42762 91175 77869 64845 29641 16848 23456 59935 7806 48755 53999 4819 22266 55601 38090 77762 65146 65912 31278 6647 36754 54109 80191 55728 28896 71471 14379 81459 16809 92471 58603 3810 271 29544 3467 90543 62543 27159 86415 97198 42304 76360 72237 20030 27786 79177 80055 54339 13304 86552 98073 16075 75528 91907 61655 8312 43279 82316 36720 85576 70192 5805 37136 99071 26198 32428 57478 283 11149 65880 59108 27200 42667 33720 47287 14389 51969 20706 1570 84045 47131 34420 54360 10538 20448 66072 69041 80120 14875 79482 21648 58626 1720 42283 21653 33330 73835 47425 80856 16540 7907 72778 92722 97371 46531 88922 11187 22422 8229 7679 15499 26343 55820 63628 90481 45049 3725 59017 65108 59541 87820 37569 74934 17319 6979 30834 36590 15454 38891 67925 75499 35289 60455 634 97717 30307 7905 60891 46804 36249 93605 49497 46053 99977 13871 79124 39195 66669 47930 35098 34050 15197 94928 85381 59928 87984 51662 97491 12658 32874 71278 41525 18509 8409 28949 92597 30336 7887 29976 45420 56771 26540 55960 6417 75009 71166 13859 555 94204 68238 52502 42277 42003 37162 42506 28588 84083 20095 64137 
11420 2273 45954 15273 36740 39339 20601 39195 5171 23025 49932 7862 9420 17678 47317 40354 19840 32865 33145 42940 12106 2383 36551 46964 24440 11578 15507 9667 17055 29026 10945 35865 10002 10795 16172 45276 10596 20468 15831 30505 44027 3301 46989 23979 2112 47319 45076 3461 33960 28518 10439 33200 42292 8870 444 2900 4787 5647 978 2577 19689 29394 45352 39183 41396 13298 48211 21122 16604 8054 14509 21049 1721 2954 24407 39688 49371 21392 9836 11113 2274 41013 48685 31664 44297 40188 31778 39388 8804 23679 11808 47098 39003 35343 17619 28366 2844 23770 47598 7846 13412 5742 25634 46870 29650 21468 1166 32106 9504 5624 23679 3499 1007 49149 24130 35415 35395 38260 46764 35562 16867 37902 15978 28925 43586 8759 24690 46004 30573 14102 5396 40812 28621 32685 14414 9239 34050 11469 13508 25210 14266 45368 40428 30140 34310 11401 31979 47742 8003 35683 48887 35791 17256 4756 45888 21364 9702 42626 7249 34707 31177 29353 8206 44355 530 12875 26014 13132 24248 13143 10063 39886 20548 3228 3212 18809 48759 30036 34643 23660 46631 46099 49544 20579 43463 16823 15031 25648 30826 1944 41890 40181 37762 24182 17717 8651 1925 6166 38059 27053 
199
1201 9097 69256 3339 92450 26755 11286 62983 41454 2418 81511 85206 30646 63195 85433 23533 6872 13795 92275 42349 2849 54311 78227 55245 84601 55055 92345 45568 27370 6876 13043 42382 33441 38679 23885 21464 47307 17109 47165 51638 17841 29395 81633 5676 49358 99635 59468 83696 80419 69722 88802 96147 77552 33953 7074 97078 12099 59227 86471 8062 34449 21091 44130 58025 59073 45628 60828 25417 84763 74672 73380 67731 15594 17551 25820 23180 25484 194 75079 45000 18425 51057 10843 94511 66821 58419 84581 10561 53835 2380 19071 28377 80424 15237 21263 53839 88185 57835 39512 627 35026 60452 7309 71018 90103 28268 12049 97840 65607 66129 83132 99007 75320 92192 68910 99966 41829 36440 172 96415 35565 29742 70292 42048 50681 444 77479 64047 22907 28003 47546 57325 48058 27524 15477 99802 39160 16666 26979 6153 53193 51231 90409 60411 23588 85113 52064 94806 60735 93122 31949 35602 66641 63427 67757 1003 33505 38703 24375 40926 40560 78969 82184 46475 91896 38597 51406 15536 85388 83634 44243 4867 96521 60132 16690 89688 52123 98677 98526 41540 99770 28775 71018 92698 40054 79749 57145 68487 14284 37783 58842 58209 74362 92823 58756 9392 49284 43363 14403 
43219 26271 38477 26434 16916 2024 22255 43597 7184 7376 49624 30803 12239 33714 36351 18162 13604 28182 5769 43170 8499 14606 14943 10713 48936 21316 27570 39377 42688 20324 30781 661 361 46171 2239 38895 32855 12986 24181 44470 22637 24405 36337 1039 21533 8607 25507 3268 23356 13348 7127 31206 26498 46559 11341 11176 13173 45762 5679 30142 44101 45688 28988 17304 15583 4890 47580 6371 24911 49174 49204 629 28187 20732 16036 17585 34659 5260 12791 10637 34170 10076 40101 22343 31521 41980 28470 41540 41521 42254 10957 37005 11628 10687 49171 22872 43141 11666 8398 37093 3536 33646 35717 14056 48082 34608 5028 35190 21007 26019 48656 48118 18234 9647 42405 27207 37833 26323 35073 40363 19972 23483 10408 1295 6058 37020 14527 27332 45202 44365 12389 47195 10228 45620 47259 6775 30010 18698 46180 41397 18646 15326 11809 17025 46345 43742 44127 42280 39325 26437 15733 49299 11028 37998 43111 14652 13040 21521 31047 39208 26025 27758 10685 12139 2320 25386 47060 11834 48221 39486 13729 37841 4899 39645 29289 20395 28271 27108 3332 15767 31612 23958 3308 6906 41332 1607 26068 18406 7476 36168 1577 32584 891 6941 30979 7362 41659 44282 45540 
