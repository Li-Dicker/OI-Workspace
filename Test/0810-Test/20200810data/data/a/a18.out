Case #1: 14966258
Case #2: 13140125
Case #3: 15277967
Case #4: 14323712
Case #5: 14061072
Case #6: 13276324
Case #7: 13669601
Case #8: 14180816
Case #9: 13698575
Case #10: 14031886
